napari/components/viewer_model.py

```diff
@@ -383,8 +383,19 @@ def _sliced_extent_world_augmented(self) -> np.ndarray:
             )
         return self.layers._extent_world_augmented[:, self.dims.displayed]
 
-    def reset_view(self) -> None:
-        """Reset the camera view."""
+    def reset_view(self, screenshot=False) -> None:
```

Comment:
Maybe this should be `margins=True`?
I can totally see wanting to get rid of the margin in general, whether screenshotting or not.

Reply:
and reverse the login below of course. `margins=True` would be current behavior, the default. and False would be the current screenshot=True.

Reply:
I think this should be `margin=0.05`, and for screenshots you would use `margin=0`.

Reply:
Oh and make it a keyword-only arg:

```suggestion
    def reset_view(self, *, margin=0.05) -> None:
```

because I have a PR open allowing layers to be passed in. 😅